Node.Path.resolve breaks the build on latest bs-platform

Calling Node.Path.resolve() breaks the build when using bs-platform 2.2.2.
```
  We've found a bug for you!
  /home/zach/src/bs-puppeteer/__tests__/puppeteer_test.re

  It's possible that your build is stale.
  Try to clean the artifacts and build again?

  Here's the original error message
  The files /home/zach/src/bs-puppeteer/node_modules/bs-node/lib/ocaml/node.cmi
  and src/Puppeteer-BsPuppeteer.cmi make inconsistent assumptions
  over interface Node

ninja: build stopped: subcommand failed.
>>>> Finish compiling(exit: 1)
```

I think this is because the type signatures differ between bucklescript's `resolve` and the one included here. BS has it as `string -> string -> string`, whereas the one here takes an array of strings using `[@@bs.splice]`.

Not sure what the best way to deal with this is. We actually only really wanted to use the Buffer module.

--- a/bsconfig.json
+++ b/bsconfig.json
@@ -1,5 +1,6 @@
 {
   "name": "bs-node",
+  "namespace": true,
   "bsc-flags": ["-bs-no-version-header", "-bs-super-errors"],
   "sources": [
     {

